test/unit/io/sam_file/format_bam_test.cpp

```diff
@@ -238,14 +238,14 @@ struct alignment_file_read<seqan3::format_bam> : public alignment_file_data
 // parametrized tests
 // ---------------------------------------------------------------------------------------------------------------------
 
-INSTANTIATE_TYPED_TEST_SUITE_P(bam, alignment_file_read, seqan3::format_bam, );
-INSTANTIATE_TYPED_TEST_SUITE_P(bam, alignment_file_write, seqan3::format_bam, );
+INSTANTIATE_TYPED_TEST_SUITE_P(bam, sam_file_read, seqan3::format_bam, );
+INSTANTIATE_TYPED_TEST_SUITE_P(bam, sam_file_write, seqan3::format_bam, );
```

Comment:
Why is there a template class definition for `sam_file_read<seqan3::format_bam>` but not for `sam_file_write<seqan3::format_bam>`?

Reply:
I don't know, I just replaced a name.

I guess that output definition is the same between sam and bam, so that it does not need a special specialisation of that class.